lib/api/model/pokemon_ability.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'pokemon_ability.freezed.dart';
+part 'pokemon_ability.g.dart';
+
+@freezed
+class PokemonAbility with _$PokemonAbility {
+  factory PokemonAbility({
```

Comment:
kindly remove Pokemon prefix - Ability is already enough 🙂 

this also applies to some other models except for `pokemon_details.dart `, just leave it as is.